frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/row_property.dart

```diff
@@ -140,7 +143,28 @@ class _PropertyCellState extends State<_PropertyCell> {
       child: SizedBox(
         width: 16,
         height: 30,
-        child: _isFieldHover ? const FlowySvg(FlowySvgs.drag_element_s) : null,
+        child: AppFlowyPopover(
+          controller: _popoverController,
```

Comment:
Don't use the same _popoverController as below. This makes the popover appear in the wrong place.